### DISCUSSION TITLE
How to use

### DISCUSSION THREAD

**Discussion by YXGuan** [lucasw/rviz_camera_stream#45] [General]
Hi Lucas @lucasw ,

Sorry I have a stupid question here, how to use this library? I am still quite new to ROS and would really appreciate your guidance.

Thanks,

**Comment by lucasw** [comment]
The first step is to run the demo:

```
roslaunch rviz_camera_stream demo.launch
```

https://github.com/lucasw/rviz_camera_stream/blob/noetic-devel/launch/demo.launch starts two camera streams, the important things it does are set up camera_infos and tf transforms that the plugin needs to know where the camera is located and what angle-of-view it has.

**Reply by YXGuan** [reply]
Awesome, thanks a lot!
